### PR TITLE
Fix for coolors.co

### PR DESCRIPTION
**Pull request by CrispyMiner**
Saturation of the color picker on coolors.co was inverted so  needed to be inverted.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3070,6 +3070,9 @@ CSS
 .generator_color.is-light {
     color: #000 !important;
 }
+.color-picker_saturation {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3073]: I'm not able to find this element.

Comment (CrispyMiner) [reply src/config/dynamic-theme-fixes.config:3073]: It's in the palette generator part of the site, click generator and it should be in there.

To view the actual color picker, do the following:
Click the hex color number
Click the drop down arrow in that menu
Select Picker
image

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3070,6 +3070,9 @@ CSS
...
+    filter: invert(100%) brightness(110%) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3074]:
```suggestion
    background-image: linear-gradient(right,rgba(255,255,255,0),#fff) !important;
```

Comment (CrispyMiner) [reply src/config/dynamic-theme-fixes.config:3074]: That didn't work for me, I think that while it's not the way to do it, it still works at least I guess?

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -3070,6 +3070,15 @@ CSS
...
     color: #000 !important;
 }
+header, div#sticky-cad_img, .page-title {
+    filter: saturate(0) brightness(50%) !important;
+}
+.sub-header {
+    filter: saturate(0) brightness(65%) !important;
+}
+.color-picker_saturation {
+    filter: invert(100%) brightness(120%) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3080]:
```suggestion
    background-image: -webkit-linear-gradient(right, rgba(255, 255, 255, 0), #fff) !important;
```

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -3070,6 +3070,15 @@ CSS
 .generator_color.is-light {
     color: #000 !important;
 }
+header, div#sticky-cad_img, .page-title {
+    filter: saturate(0) brightness(50%) !important;
+}
+.sub-header {
+    filter: saturate(0) brightness(65%) !important;
+}
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3078]: The elements already looks fine to me?

**Pull request closed without merging**